cmd/thanos/query.go

```diff
@@ -193,6 +192,7 @@ func registerQuery(m map[string]setupFunc, app *kingpin.Application) {
 			time.Duration(*instantDefaultMaxSourceResolution),
 			*strictStores,
 			component.Query,
+			time.Duration(*defaultEvaluationInterval),
```

Comment:
This is duplicate with https://github.com/thanos-io/thanos/pull/2876/files#diff-fdba606ba45878a31172c82a2b017303R178, please remove it.